### ISSUE TITLE
SYSU dataset accuracy

### ISSUE THREAD

**Issue by ChengxiHAN** [wgcban/ChangeFormer#48]
hi my friend, i try to use SYSU dataset to train, but I  got error results as follow:

acc: 1.00000 miou: 0.50000 mf1: 0.50000 iou_0: 1.00000 iou_1: 0.00000 F1_0: 1.00000 F1_1: 0.00000 precision_0: 1.00000 precision_1: 0.00000 recall_0: 1.00000 rec    all_1: 0.00000

could you help me tell me the reasons? many thanks

**Comment by wgcban** [comment]
@ChengxiHAN That is weird. How large is the training dataset? Is this after the training?

**Comment by ChengxiHAN** [comment]
> 

image size=256*256， train/val/test=12000/400/400. that is the training process result.

**Comment by ChengxiHAN** [comment]
> actually, the cdd, whu,levir, dsifn datasets are all ok.

**Comment by wgcban** [comment]
@ChengxiHAN Have you tried using a pre-trained model on one of the above datasets as an initialization for your model? How did the training loss converge during the training process? What was the training accuracy achieved? Was it improving over the course of training? If not, it could indicate an issue with the data loading or the dataset itself.
